CPAC/utils/interfaces/netcorr.py

```diff
@@ -0,0 +1,217 @@
+"""Interface for AFNI 3dNetCorr"""
+import glob
+import os
+from nipype.interfaces.afni.base import AFNICommand, AFNICommandInputSpec, \
+                                        AFNICommandOutputSpec
+from nipype.interfaces.base import File, isdefined
+from traits.api import Bool, List
+
+
+class NetCorrInputSpec(AFNICommandInputSpec):
+    timeseries = File(
+        desc="input time series file (4D data set)",
+        exists=True,
+        argstr="-inset %s",
+        mandatory=True,
+    )
+    parcellation = File(
+        desc="input set of ROIs, each labelled with distinct integers",
+        exists=True,
+        argstr="-in_rois %s",
+        mandatory=True,
+    )
+    mask = File(
+        desc="can include a whole brain mask within which to "
+        "calculate correlation. Otherwise, data should be "
+        "masked already",
+        exists=True,
+        argstr="-mask %s",
+    )
+    automask_off = Bool(
+        True,
+        desc='If you want to neither put in a mask '
+             '*nor* have the automasking occur',
+        argstr='-automask_off', usedefault=True
+    )
```

Comment:
I added this option, but I think we don't need it: https://github.com/FCP-INDI/C-PAC/pull/1595#issuecomment-963560788